opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -95,9 +95,6 @@
 TELEMETRY_SDK_NAME = "telemetry.sdk.name"
 TELEMETRY_SDK_VERSION = "telemetry.sdk.version"
 
-OPENTELEMETRY_SDK_VERSION = pkg_resources.get_distribution(
```

Comment:
Would [opencensus-exporter](https://github.com/open-telemetry/opentelemetry-python/blob/master/exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/util.py#L29) have the same issue?

Reply:
I believe it would if the autodoc directive included that `util.py` file, but it just includes it doesn't: https://github.com/open-telemetry/opentelemetry-python/blob/master/docs/exporter/opencensus/opencensus.rst

So all you see in the docs [here](https://opentelemetry-python.readthedocs.io/en/stable/exporter/opencensus/opencensus.html#) is the docstring here: https://github.com/open-telemetry/opentelemetry-python/blob/b8a8016e3e5ff319a5224d2ecfa343943bcbd740/exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/__init__.py#L15-L18

Reply:
I could fix that one up to do this as well though in case it's added in the future. Another option is to update the `docs-requirements.txt` file to actually install the opentelemetry-sdk/opentelemetry-api in the virtualenv so that `pkg_resources.get_distrubution()` works correctly.

Reply:
Gonna go with this second approach instead 👍